dkg/frost/frost.go

```diff
@@ -32,6 +36,8 @@ type FROST struct {
 	state   *ProtocolState
 }
 
+// ProtocolConfig contains properties needed to start the protocol like requestID,
+// operatorID, threshold, operator list etc.
 type ProtocolConfig struct {
```

Comment:
The name sounds confusing. Config refers to more infra stuff (i.e. Network, Program config). Let's call it `InstanceParams` if we don't have a better name.